### DISCUSSION TITLE
리팩터링 8주차 안건 (221002-221008)

### DISCUSSION THREAD

**Discussion by punchdrunkard** [prgrms-web-devcourse/Team-BackFro-ArtZip-FE#306] [02.Refactoring]
.

**Comment by punchdrunkard** [comment]
![image](https://user-images.githubusercontent.com/74234333/193420915-510e69c8-b8e6-4eb0-bcfe-0c7e497190eb.png)
![image](https://user-images.githubusercontent.com/74234333/193420926-9e11ed05-7e19-497b-b173-ec61f5fe5fe5.png)

@gitul0515 
오늘 날짜 이후로 날짜를 선택했을 때
글 작성 버튼 까지 누르고 나서야 경고 메시지를 보고 다시 날짜를 선택하는 과정이 불편하게 느껴졌습니다.
날짜를 선택할 때, 오늘 날짜 이후의 날짜는 선택 자체를 불가능하게 할 수 있나요?

**Reply by gitul0515** [reply]
antd 속성 확인해보기

**Comment by punchdrunkard** [comment]
![preview](https://user-images.githubusercontent.com/74234333/193430493-d22ca78f-a23a-4927-a587-792df4553290.gif)

동그랗게 어두워졌으면 좋겠습니다!

**Reply by punchdrunkard** [reply]
또한, 헤더의 유저 아바타에서도 호버 시 어두워지는 효과가 있으면 어떨까요?

**Reply by punchdrunkard** [reply]
![image](https://user-images.githubusercontent.com/74234333/193449329-bedca7f1-f191-4dad-9cfd-289b586b843a.png)

여기 수정 부분도 어두워지면서 수정이란 글씨가 뜨면 좋을 것 같습니다!

**Reply by punchdrunkard** [reply]
![image](https://user-images.githubusercontent.com/74234333/193449354-27d5079c-c3bf-4d1d-9dea-29b3fd5b564f.png)

**Comment by punchdrunkard** [comment]
![image](https://user-images.githubusercontent.com/74234333/193430709-3196342a-5849-4749-a85a-400075a2a7e1.png)
페이지 별 기능 작성

**Comment by gitul0515** [comment]
커뮤니티, 후기 상세 - SSR 또는 SSG(ISR) 적용

**Reply by punchdrunkard** [reply]
ISR 은 적용이 불가능 했습니다 ㅠㅠ 왜냐하면 토큰을 첨부해서 api 호출을 해야 해서, ISR(SSG)에서는 쿠키를 확인할 수가 없어요!

**Comment by gitul0515** [comment]
스켈레톤 UI

**Reply by gitul0515** [reply]
ExhibitionCard ReviewCard

**Reply by gitul0515** [reply]
CLS라는 성능 지표 향상

**Comment by gitul0515** [comment]
후기 상세 content에서 ' '

**Comment by gitul0515** [comment]
로그아웃 시 낙관적 업데이트 
서버사이드 렌더링 로딩 중 처리

**Reply by gitul0515** [reply]
Next.js에서 지원하는 기능이 있다.

**Reply by punchdrunkard** [reply]
정확히는 router 에서 페이지 전환마다 컴포넌트를 보여주는 방식이더군요! 곧 PR 올리겠습니다!

**Comment by punchdrunkard** [comment]
쿠키의 보안 문제 #310

**Comment by punchdrunkard** [comment]
모든 페이지에 대해서 쿠키를 조작할 때 #309

**Comment by gitul0515** [comment]
React Suspense

**Comment by gitul0515** [comment]
스냅샷 또는 시연 영상
